src/slack.js

```diff
@@ -102,6 +102,19 @@ function confirmOrder(ts) {
   );
 }
 
+function removeConfirmation(ts) {
+  slack.web.reactions.remove(
+    config.orderReaction,
+    {channel: config.slack.lunchChannelId, timestamp: ts}
+  );
+}
+
+/**
+ * @param {string} key - order (parent) timestamp
+ * @param {string} value - unknown order message timestamp
+ */
+const unknownOrderMessages = {};
```

Comment:
Can you do this without locally storing sent messages?
This cache is never cleared.